### PR TITLE
HooksTest: Fix using class Block

### PR DESCRIPTION
**Pull request by paladox**
Since 1.38, it has been namespaced under MediaWiki\ Block.

## Summary by CodeRabbit

- **Tests**
	- Updated mock object creation in specific test methods to align with the new namespace structure.

### CODE REVIEW

**Comment by paladox** [comment]
@coderabbitai full review

File: tests/phpunit/MediaWiki/HooksTest.php [hunk 1]
```diff
@@ -1389,7 +1389,7 @@ public function callSoftwareInfo( $instance ) {
 	public function callBlockIpComplete( $instance ) {
 		$handler = 'BlockIpComplete';
 
-		$block = $this->getMockBuilder( '\Block' )
+		$block = $this->getMockBuilder( '\MediaWiki\Block\Block' )
```
Comment (JeroenDeDauw) [inline tests/phpunit/MediaWiki/HooksTest.php:1392]: This is better:

`$block = $this->createMock( Block::class );`

(with an import statement at the top of file)

Comment (paladox) [reply tests/phpunit/MediaWiki/HooksTest.php]: Thanks, done.

**Comment by paladox** [comment]
@coderabbitai full review

**Comment by paladox** [comment]
Hmm it fails

**Comment by paladox** [comment]
> Hmm it fails

Think I fixed it.

**Pull request merged**